src/registrar/tests/test_copy_names_from_contacts_to_users.py

```diff
@@ -18,8 +18,8 @@ def setUp(self):
         self.user2 = User.objects.create(username="user2")
         self.user3 = User.objects.create(username="user3")
         self.user4 = User.objects.create(username="user4")
-        # The last user created triggers the creation of a contact and attaches itself to it. @Neil wth is going on?
-        # This bs_user defuses that situation so we can test the code.
+        # The last user created triggers the creation of a contact and attaches itself to it. See signals.
```

Comment:
**question (non-blocking)**: I don't see how this is relevant to this PR? Did this come over from another branch?

Reply:
I remembered that I'd left the @Neil comment in the previous PR and thought it's ok to sneak in a comment clean up